interfaces/builtin/upower_observe.go

```diff
@@ -100,7 +100,7 @@ dbus (send)
     bus=system
     path=/org/freedesktop/login1{,/**}
     interface=org.freedesktop.login1.Manager
-    member={CanPowerOff,CanSuspend,CanHibernate,CanHybridSleep,PowerOff,Suspend,Hibernate,HybridSleep,Inhibit}
+    member={CanPowerOff,CanSuspend,CanHibernate,CanSuspendThenHibernate,CanHybridSleep,PowerOff,Suspend,Hibernate,SuspendThenHybernate,HybridSleep,Inhibit}
```

Comment:
You added `ListInhibiters` to the shutdown interface. Would it make sense here too, given that it also allows `Inhibit`?

Reply:
Makes sense, just added it.